uber/api.py

```diff
@@ -61,3 +63,43 @@ def lookup(self, location):
             return [job.to_dict(job_fields) for job in session.query(Job).filter_by(location=label_lookup[location]).all()]
 
 services.register(JobLookup(), 'shifts')
+
+
+class DepartmentLookup:
+    def list(self):
+        with Session() as session:
+            output = {}
+            for dept in c.JOB_LOCATION_VARS:
+                output[dept] = dict(c.JOB_LOCATION_OPTS)[getattr(c, dept)]
+            return output
+
+services.register(DepartmentLookup(), 'dept')
+
+event_fields = [
+    'EVENT_NAME',
+    'ORGANIZATION_NAME',
+    'YEAR',
+    'EPOCH',
+
+    'EVENT_VENUE',
+    'EVENT_VENUE_ADDRESS',
+
+    'AT_THE_CON',
+    'POST_CON',
+]
+
+
+class EventLookup:
+    def info(self):
+        output = {
+            'API_VERSION': __version__
+        }
+        for field in event_fields:
+            output[field] = getattr(c, field)
+        return output
+
+    def lookup(self, field):
+        if field.upper() in event_fields:
+            return getattr(c, field.upper())
+
+services.register(EventLookup(), 'event')
```

Comment:
hey so I get that these are supposed to be event-specific config params, but it might be better to just name this what it is, which is something like ConfigLookup.  over time people are just going to keep tacking stuff onto this because it's easy and it won't be event-specific anymore.